src/witan/models/dem/ccm/mort/mortality.clj

```diff
@@ -120,3 +120,19 @@
    (cf/project-component-fixed-rates population-at-risk
                                      initial-projected-mortality-rates
                                      :death-rate :deaths)})
+
+(defworkflowfn project-deaths
+  ""
```

Comment:
add doc string "Takes the current year of the projection, a dataset with population at risk from that year, and another dataset with death rates for the population for all projection years. Death rates are filtered for the current year. Returns a dataset with a column of deaths, which are the product of popn at risk & death rates"